package.json

```diff
@@ -7,16 +7,22 @@
     "@emotion/styled": "^11.10.4",
     "@mui/icons-material": "^5.10.9",
     "@mui/material": "^5.10.10",
+    "@reduxjs/toolkit": "^1.8.6",
```

Comment:
- [ ] Your app has many eslint errors and react warnings.  So much, it's impossible to run your frontend app without deactivating ESlint.

Please, kindly fix those eslint and React errors in order to make your app work as intended. 👍